src/course4/homework/Challenge5b.java

```diff
@@ -12,21 +12,16 @@ public static void main(String[] args) {
 
         System.out.println("Introduce your number:");
         double number = scan.nextDouble();
-        boolean trueValue = true;
 
-        // next we have two "if" loops to verify if your number is higher, lower or equal to 0
-        if (number != 0) { //this verifies if the number is different than 0;
-
-            trueValue = (number > 0); //next step if the number is different than 0 (I choose higher out of habit;
-                                      // it can also be lower;
+        //we have two "if" loops to verify if your number is higher, lower or equal to 0
```

Comment:
just FYI: `if` is not a loop, but a conditional statement.

Reply:
my bad. I named it "loop" out of habit, but I renamed it now.